packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -989,6 +989,10 @@ export abstract class FluidDataStoreContext
 			);
 	}
 
+	public deleteChildSummarizerNodeFn(id: string) {
```

Comment:
I'd remove "Fn" suffix. getCreateChildSummarizerNodeFn returns a function, thus it has Fn at the end. This one does not.

Reply:
yes, I had kept it because inside the constructor from dataStore.ts, they do the same thing and named it deleteChildSummarizerNodeFn :) 

I can remove the Fn